cas-server-core-logging/src/main/java/org/jasig/cas/util/CasLoggerContextInitializer.java

```diff
@@ -1,10 +1,7 @@
 package org.jasig.cas.util;
 
-import org.apache.commons.lang3.StringUtils;
-import org.reflections.Reflections;
-import org.reflections.scanners.SubTypesScanner;
-import org.reflections.util.ClasspathHelper;
-import org.reflections.util.ConfigurationBuilder;
+import org.apache.logging.log4j.LogManager;
```

Comment:
I can switch to reflection here so it gracefully fails if the deployer removes log4j, the only issue would be the need for explicitly definingq the listener in web.xml. I tried relying on the servlet 3 auto config, but it still shutdown log4j too soon.